app/views/support_interface/application_forms/show.html.erb

```diff
@@ -54,7 +54,9 @@
 <% end %>
 
 <% if @application_form.application_choices.any?(&:awaiting_references?) %>
-  <%= govuk_button_link_to 'Make changes to courses', support_interface_change_course_path(@application_form) %>
+<div class='govuk-body'>
+  <%= govuk_link_to 'Add or withdraw course choices', support_interface_change_course_path(@application_form) %>
```

Comment:
I liked [this tweet](https://mobile.twitter.com/thetuttingtutor/status/1252710968817319936) the other day:

> General UX writing rule I follow:
> • Navigation = nouns
> • Buttons = verbs

This could be a button as it was before, but yeah, minor.